app/views/tag/_tagging.html.erb

```diff
@@ -48,15 +48,22 @@ $(function () {
 <% parent ||= nil %>
 
 <% if current_user && (parent != :profile || (current_user.id == user.id || logged_in_as(['admin']))) && !@preview  %>
-  <a data-toggle="tooltip" title="Add tags/View all" aria-label="Add tags/View all" id="tags-open" class="btn btn-circle btn-circle-sm tags-open" style="float:left;">
-    <i class="fa fa-plus" style="color:#808080;"></i>
-  </a>
+  <% if current_user.first_time_poster && !(current_user.username == @node.author.username || ( @node.coauthors && @node.coauthors.exists?(username: current_user.username)) || logged_in_as(['admin', 'moderator'])) %>
```

Comment:
@17sushmita This line is giving me errors in my local env. When I log in as a user who has not created any posts and then try to visit my profile page, it displays this:

<img width="1017" alt="Screenshot 2021-07-05 at 15 16 34" src="https://user-images.githubusercontent.com/63427719/124484986-fed96580-dda3-11eb-95f1-888937175338.png">

I think the issue is that since the user has no posts then `@node` is null. Can you try it on your end and see if you have the same issue? Thanks!

It's the same on https://stable.publiclab.org/ too

<img width="1280" alt="Screenshot 2021-07-05 at 15 21 42" src="https://user-images.githubusercontent.com/63427719/124485664-b8383b00-dda4-11eb-9284-5d51d9ef23de.png">

Reply:
Can you provide more information regarding this? I wasn't able to reproduce this error. I created a new account on publiclab which had no posts and visited your profile and it is accessible.
![image](https://user-images.githubusercontent.com/42088159/124504697-9f03ae80-dde5-11eb-9402-cd67571fad90.png)

Reply:
@17sushmita Try visiting the profile of the newly created user while logged into that account.

Reply:
I just created this user [1700sushmita]() and still working fine🧐
![image](https://user-images.githubusercontent.com/42088159/124512193-6c61b200-ddf5-11eb-80f8-a81dce515c8e.png)
also from new user to the previous account-
![image](https://user-images.githubusercontent.com/42088159/124512241-83a09f80-ddf5-11eb-84e8-12da0746b7d4.png)

Reply:
Okay, got the error while visiting the new user's own profile.